python/pylibcudf/pylibcudf/tests/test_contiguous_split.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pylibcudf as plc
+import pytest
+from utils import assert_table_eq
+
+mixed_pyarrow_tables = [
+    pa.table([]),
+    pa.table({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]}),
+    pa.table({"a": [1, 2, 3]}),
+    pa.table({"a": [1], "b": [2], "c": [3]}),
+    pa.table({"a": ["a", "bb", "ccc"]}),
```

Comment:
```suggestion
    pa.table({"a": ["a", "bb", "ccc"]}),
    pa.table({"a": [1, 2, None], "b": [None, 3, 4]})
    pa.table({"a": [["a", "b"], ["cde"]], "b": {"alpha": [1, 2], "beta": None}})
```

Can we test with nullable columns and nested types as well?